calico_kubernetes/calico_kubernetes.py

```diff
@@ -604,12 +689,9 @@ def _label_to_tag(self, label_key, label_value):
 
 
 if __name__ == '__main__':
-    configure_logger(logger)
-    logger.info('Args: %s' % sys.argv)
-    logger.info("Using ETCD_AUTHORITY=%s", os.environ[ETCD_AUTHORITY_ENV])
-    logger.info("Using CALICOCTL_PATH=%s", CALICOCTL_PATH)
-    logger.info("Using KUBE_API_ROOT=%s", KUBE_API_ROOT)
-    logger.info("Using DEFAULT_POLICY=%s", DEFAULT_POLICY)
+    configure_logger(logger, LOG_LEVEL.upper(), True)
```

Comment:
You could simplify this by moving the .upper() call above to where you are reading the environment variable.

So,
```
LOG_LEVEL = env.get(...).upper()
```